src/usdb_syncer/json.py

```diff
@@ -0,0 +1,114 @@
+"""Generates a JSON from the passed song list."""
```

Comment:
Perhaps the module name should be changed to avoid name conflicts with the module from the stanard library.

Reply:
I think `json_export.py` makes sense or just `export.py` if we merge with `pdf.py`

Reply:
Since pdf.py will be removed for now, json_export.py sounds good. Also export.py is all fun and games until you want to add the import equivalent. 😄